lib/commands/build.js

```diff
@@ -10,9 +9,9 @@ module.exports = Command.extend({
   aliases: ['b'],
 
   availableOptions: [
-    { name: 'environment', type: String,  default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
-    { name: 'output-path', type: path,    default: 'dist/',       aliases: ['o'] },
-    { name: 'watch',       type: Boolean, default: false,         aliases: ['w'] },
+    { name: 'environment', type: String,             default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
+    { name: 'output-path', type: function Path() {}, default: 'dist/',       aliases: ['o'] },
```

Comment:
that what I meant with:

> did path have any hidden meaning when validating the command line option?

I'm not quite convinced yet that this is the right way to go. If we move forward in this direction we should at least extract the `Path` thing into a utility module/file.